src/pages/blocks/components/FilterModal.tsx

```diff
@@ -155,4 +156,13 @@ const FilterModal: React.FC<FilterModalProps> = ({
   );
 };
 
+export async function getStaticProps({ locale }: any) {
+  return {
```

Comment:
@Aar-if we've separate components folder, please avoid creating separate folder for components, if u want u can create subfolder inside components, name it as blocks and move the this compoent there

Reply:
@Aar-if for components we dont require serverSidetranslations

Reply:
Okay